app/_data/docs_nav_kuma_dev.yml

```diff
@@ -19,13 +19,95 @@ items:
             url: "/introduction/how-kuma-works/#vm-and-k8s-support"
           - text: Kuma vs XYZ
             url: "/introduction/how-kuma-works/#kuma-vs-xyz"
-      - text: Deployments
-        url: /introduction/deployments/
       - text: Enterprise
         url: /introduction/enterprise/
       - text: Changelog
         absolute_url: true
         url: https://github.com/kumahq/kuma/blob/master/CHANGELOG.md
+  - title: Kuma in Production
+    group: true
+    items:
+      - title: Deployment topologies
+        group: true
+        items:
+          - text: Overview
+            url: /production/deployment/
+          - text: Standalone deployment
+            url: /production/deployment/stand-alone/
+          - text: Multi-zone deployment
+            url: /production/deployment/multi-zone/
+          - text: Default ports
+            url: /production/deployment/networking/
+      - title: Control plane deployment
+        group: true
+        items:
+          # - text: Kong Mesh license
+            # url: /production/
+          - text: Deploy a multi-zone global control plane
+            url: /production/cp-deployment/multi-zone/ 
+            group: true
+            items:
+              # - text: Add a zone
+                # url: /production/
+              # - text: Configure multi-zone authentication
+                # url: /production/
```

Comment:
@lahabana @johnharris85 I didn't have a link for these in the IA spreadsheet so I wasn't sure if we'd be creating these pages from scratch or if there were existing pages that cover these two topics.

Reply:
I think we'd do it from scratch yes